src/Components/Navbar/navbar.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Navbar = () => {
+  return (
+    <header
+      className="header cont-shadow"
+      style={{ width: "96rem", backgroundColor: "white" }}
+    >
+      <div className="logos">
+        <Link to="/">
+          <a>
+            W <span className="logos-o">O</span>W Quiz
+            <span className="logos-excla">!</span>
+          </a>
+        </Link>
+      </div>
+      <ul className="header-nav">
+        <li style={{ marginTop: "0.25rem", cursor: "pointer" }}>
+          <li style={{ cursor: "pointer" }}>
+            <a>
+              <i
+                class="fas fa-sign-out-alt"
+                style={{ marginRight: "0.5rem" }}
+              ></i>
+              Github
```

Comment:
Suggestion: Add GitHub link to your in this

Reply:
Okayy.